### ISSUE TITLE
Ship Area of Interest - Questions

### ISSUE THREAD

**Issue by lfunkhouser** [Esri/joint-military-symbology-xml#522]
Symbol Set Code: 25
Code: 200400

Ship Area of Interest requires one anchor point, defining the center.
Is this an area or a point symbol?
Is there a set number of circles within the Ship Area of Interest? Both the template and the example is shown as 20 circles.  
If area, is the shape always a rectangle and should there be additional attributes that determine the size of the symbol (i.e. length and width)?

**Comment by joebayles** [comment]
@lfunkhouser, 
- Is this an area or a point symbol?
_These are area symbols._
- Is there a set number of circles within the Ship Area of Interest? Both the template and the example is shown as 20 circles.
_No. The template should be empty, and the example is simply demonstrating how it could look._
- If area, is the shape always a rectangle and should there be additional attributes that determine the size of the symbol (i.e. length and width)?
_No, the shape is not always a rectangle. There probably should be additional attributes for size._

Basically, a ship will have an Area of Interest, shaped either as an ellipse or circle (...401), or a rectangle (...402). Inside that Area of Interest could be 1-n Maneuver Areas (...400), that are circles. All of these are dynamic polygons.

Active Maneuver Area (...500), is currently listed as a static symbol, to be drawn around the Own Ship Icon (30150000), but it's my opinion that this should be dynamic.

The USN and USMC will review this question and possibly submit change proposals.

FYI @wmcgrane
